modules/EvManager/main/car_simulation.hpp

```diff
@@ -98,8 +110,17 @@ class CarSimulation {
 
 private:
     SimulationData sim_data;
+    const module::Conf& config;
+    std::chrono::time_point<std::chrono::steady_clock> timepoint_last_update;
+    double charge_current_a{0};
+    double charge_voltage_v{0};
```

Comment:
Is there a reason this is a class member? As far as I can see this is only used in one function where it's just copying the value of the config so it could live only there

Reply:
changed to use the configured voltage for AC and target voltage for DC